src/clr-angular/utils/focus/focus.service.ts

```diff
@@ -45,31 +45,29 @@ export class FocusService {
   }
 
   moveTo(item: FocusableItem) {
-    this.renderer.setAttribute(this.container, 'aria-activedescendant', item.id);
     if (this.current) {
       this.current.blur();
     }
     item.focus();
     this._current = item;
   }
 
-  /**
-   * The second parameter, optional, is here to allow recursion to skip disabled items.
-   */
-  move(direction: ArrowKeyDirection, current = this.current) {
-    if (current) {
-      const next = current[direction];
+  move(event: any, direction: ArrowKeyDirection) {
+    if (this.current) {
+      // We want to prevent default behavior that results from the keydown,
+      // which may undesirably move the cursor around when using a screen reader
+      if (event) {
+        event.preventDefault();
+      }
+
+      const next = this.current[direction];
       if (next) {
         // Turning the value into an Observable isn't great, but it's the fastest way to avoid code duplication.
         // If performance ever matters for this, we can refactor using additional private methods.
         const nextObs = isObservable(next) ? next : of(next);
         nextObs.subscribe(item => {
-          if (item.disabled) {
-            return this.move(direction, item);
-          } else {
-            this.moveTo(item);
-            return true;
-          }
+          this.moveTo(item);
+          return true;
```

Comment:
Disabled items now need to be focusable so we are removing that conditional check to see if it's not disabled before moving focus.